bin/alluxio-start.sh

```diff
@@ -50,7 +51,20 @@ check_mount_mode() {
   case "$1" in
     Mount);;
     SudoMount);;
-    NoMount);;
+    NoMount)
+      if [[ "${ALLUXIO_RAM_FOLDER}" =~ ^"/dev/shm"\/{0,1}$ ]]; then
+        if [[ $( uname -a) == Darwin* ]]; then
+          # Assuming Max OS X
+          echo "ERROR: tmpFS is only enabled in Linux."
+          echo -e "${USAGE}"
+          exit 1
+        fi
+        echo "Warning: Alluxio is running on tmpfs that supports swapping."
+        echo "Warning: Check vmstat if Alluxio is slow."
+      else
+        echo "WARNING: using NoMount but ALLUXIO_RAM_FOLDER is not set to /dev/shm."
```

Comment:
Are you asking whether  ALLUXIO_RAM_FOLDER can be set to /mnt/ramdisk here? I am not sure I understand you question.